### PR TITLE
bug-1197962-add-adgroup-categories

### PR DESCRIPTION
**Pull request by tspurway**
This PR adds the adgroup_categories table migration and model.  It is destined for the master branch, and *should* be the last 
change on the production database.

r? @ncloudioj

### CODE REVIEW

**Comment by ncloudioj** [comment]
Looks good to me. r+

Let's also add a `title_bg_color` column to the `tiles` table. Like `bg-color`, it's a VARCHAR(16) with `nullable=False`. After this change, I think we're good to freeze the database changes for the master branch.

**Comment by tspurway** [comment]
I just broke the tests.  Fixing.....

**Comment by tspurway** [comment]
Also, Nan, shouldn't nullable=True, to maintain backward compatibility?  What does firefox do if it gets a payload without titleBgColor?

**Comment by oyiptong** [comment]
It should work just fine without

**Comment by tspurway** [comment]
Yeah, there doesn't seem to be a compelling reason to make it mandatory on the ingest format.

**Comment by ncloudioj** [comment]
@tspurway Yes, you're right. It's optional.

**Comment by oyiptong** [comment]
works just fine without it: https://hg.mozilla.org/integration/fx-team/rev/9d35eca4bc3a#l1.17

**Comment by tspurway** [comment]
let's try another r? @ncloudioj

File: tests/test_ingest.py [hunk 1]
```diff
@@ -501,6 +501,30 @@ def test_adgroups_channel_id_uniqueness(self):
         c = self.env.db.session.query(Adgroup).count()
         assert_equal(32, c)
 
+    def test_title_bg_color(self):
+        """
+        A simple test of frequency caps
```
Comment (ncloudioj) [inline tests/test_ingest.py]: A simple test of title_bg_color

File: tests/base.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from splice.webapp import create_webapp
 from flask.ext.testing import TestCase
 
-db_uri = os.environ.get('TEST_DB_URI') or 'postgres://localhost/splice_test'
+db_uri = os.environ.get('TEST_DB_URI') or 'postgres://tspurway:tacos@127.0.0.1/splice_test'
```
Comment (ncloudioj) [inline tests/base.py]: Oops...

**Comment by ncloudioj** [comment]
r+ @tspurway

**Pull request closed without merging**